src/Microplate/Scalars/Column2x16Well.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\Microplate\Scalars;
+
+class Column2x16Well extends AbstractColumnWell
```

Comment:
```suggestion
class Column2 extends IntRange
```

Wenn wir die Typen für Row und Column einfach anhand der Anzahl benennen, können wir sie wiederverwenden.